docs/source/task-launch.rst

```diff
@@ -163,6 +244,6 @@ processing slots the following actions occur whenever a task invokes
 Because of this behavior you can happily launch long running control tasks that
 manage worker-side clients happily, without fear of deadlocking the cluster.
```

Comment:
But this is also true of the secede/rejoin and dask.compute approaches above, no?